main/broadcasters.c

```diff
@@ -763,12 +763,90 @@ static broadcaster_ops_t atc1441_temp_hum_ops = {
     .metadata_get = atc1441_temp_hum_metadata_get,
 };
 
+/* PVVX Firmware for the Xiaomi Thermometer LYWSD03MMC Temperature and
+ * Humidity Sensor, see: https://github.com/pvvx/ATC_MiThermometer */
+
+#define PVVX_TEMP_HUM_SERVICE_UUID 0x181A
+
+typedef struct {
+//    uint16_t not_used;
```

Comment:
Please remove if this really isn't needed

Reply:
Good call. Done.